buildlib/pr/main.yml

```diff
@@ -138,9 +138,6 @@ resources:
     - container: ubuntu20_cuda_11_6
       image: nvidia/cuda:11.6.0-devel-ubuntu20.04
       options: $(DOCKER_OPT_ARGS) $(DOCKER_OPT_GPU)
-    - container: centos7_cuda11
-      image: rdmz-harbor.rdmz.labs.mlnx/ucx/centos7-mofed5-cuda11:1
-      options: $(DOCKER_OPT_ARGS) $(DOCKER_OPT_VOLUMES) $(DOCKER_OPT_GPU)
```

Comment:
I'm not familiar with CI.
Line 16-17 use different image and docker options.
Will it affect the test cases and result after removing 141-143?

Reply:
Got it